packages/cli/src/databases/repositories/sharedWorkflow.repository.ts

```diff
@@ -125,4 +125,20 @@ export class SharedWorkflowRepository extends Repository<SharedWorkflow> {
 
 		return transaction.save(newSharedWorkflows);
 	}
+
+	async findWithFields(candidateIds: string[], { fields }: { fields: string[] }) {
```

Comment:
I believe `candidateIds` is terminology that was used where this moved from? Something like `workflowIds` might make more sense here.

Reply:
Renamed!